File: module5/case-study/src/app/model/customer/customer.ts
import {CustomerType} from "./customerType";

export interface Customer {
  id:number;
  customerTypeId?: CustomerType;
  name?:string;
  dateOfBirth?:string;
  idCard?:string;
  phoneNumber?:string;
  gender?:boolean;
  email?:string;
  address?:string;
}

File: sprint 2/be/src/main/java/com/be/model/Customer.java
package com.be.model;

import javax.persistence.*;
import java.util.Set;

@Entity
public class Customer {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String name;
    private String phoneNumber;
    private boolean flagDelete;
    private String birthday;
    private String address;
    private String email;
    @OneToOne()
    @JoinColumn(name = "account_id", referencedColumnName = "id")
    private Account account;
    @OneToMany(mappedBy = "customer")
    public Set<Oder> oderSet;

    public Customer() {
    }
}
